File: Python/program74.py
# Input : 4
# Output: * * * *

def Display(iNo):
    for i in range(1,iNo + 1,1):
        print("*", end=" ")
    print()
    
def main():
    print("Enter number")
    iValue = int (input())
    
    Display(iValue)
    
    
if __name__ == "__main__":
    main()

File: C++/program74.cpp
// Input : 4
// Output: -4 -3 -2 -1 0


#include<iostream>
using namespace std;

void Display(int iNo)
{
    if(iNo < 0)
    {
        iNo = -iNo;
    }
    int iCnt = 0;
    
    for(iCnt = -iNo; iCnt <= 0; iCnt++) 
    {
        cout<<iCnt<<'\t';
    }
    
    cout<<"\n";

}

int main()
{
    int iValue = 0;

    cout<<"Enter number "<<'\n';
    cin>>iValue;


    Display(iValue);

    return 0;
}